base/sparse/sparsematrix.jl

```diff
@@ -894,12 +894,12 @@ function sprand_IJ(r::AbstractRNG, m::Integer, n::Integer, density::AbstractFloa
 end
 
 """
-    sprand([rng],m,[n],p::AbstractFloat,[rfn])
+    sprand([rng],[type],m,[n],p::AbstractFloat,[rfn])
 
 Create a random length `m` sparse vector or `m` by `n` sparse matrix, in
 which the probability of any element being nonzero is independently given by
 `p` (and hence the mean density of nonzeros is also exactly `p`). Nonzero
-values are sampled from the distribution specified by `rfn`. The uniform
+values are sampled from the distribution specified by `rfn` and have the type `type`. The uniform
```

Comment:
yeah `make docs` is the more precise way that will also rerun bootstrap to get the updated docstring into the system image